core/src/main/scala/za/co/absa/spline/harvester/conf/DefaultSplineConfigurer.scala

```diff
@@ -95,14 +100,17 @@ class DefaultSplineConfigurer(sparkSession: SparkSession, userConfiguration: Con
   protected def ignoredWriteDetectionStrategy: IgnoredWriteDetectionStrategy = instantiate[IgnoredWriteDetectionStrategy](
     configuration.getRequiredString(IgnoreWriteDetectionStrategyClass))
 
+  protected def postProcessingFilters: Seq[Filter] = configuration.getRequiredString(postProcessingFilterClasses)
+    .split(";").map(_.trim).map(instantiate[Filter])
+
```

Comment:
1. A method chain is much easier to read/debug/maintain when arranged one method per line.
1. Method `split()` doesn't de-duplicate delimiters and can return empty strings. Hence an additional `filter(_.nonEmpty)` is required.
1. Usually in Java a comma is widely used as a value delimiter. If done so then the method `getRequiredStringArray()` could be used instead of explicit splitting and trimming.

```suggestion
  protected def postProcessingFilters: Seq[Filter] =
    configuration
      .getRequiredStringArray(postProcessingFilterClasses)
      .filter(_.nonEmpty)
      .map(instantiate[Filter])
```